csharp/Microsoft.Azure.Devices.Client.NetMF/Common/Exceptions/IotHubException.cs

```diff
@@ -0,0 +1,75 @@
+// Copyright (c) Microsoft. All rights reserved.
```

Comment:
Let's attempt to use links to the source code files as we did in the UWP project. Having #if for NET_MF to control the code that you do not wish to compile is fine.